src/main/java/com/royalrangers/service/UserService.java

```diff
@@ -394,12 +392,10 @@ private User getPlatoonAdmin(User user) {
         Long platoonId = user.getPlatoon().getId();
         List<User> usersByPlatoon = userRepository.findAllByPlatoonId(platoonId);
         Optional<User> admin = usersByPlatoon.stream()
-                .filter(element -> user.getAuthorities()
-                        .contains(ROLE_ADMIN))
+                .filter(element -> user.isUserHasRole(ROLE_ADMIN))
                 .findFirst();
```

Comment:
This method should return list of admins. And then send en email to all of them.